CodePush.js

```diff
@@ -4,6 +4,9 @@ import requestFetchAdapter from "./request-fetch-adapter";
 import { AppState, Platform } from "react-native";
 import log from "./logging";
 import hoistStatics from 'hoist-non-react-statics';
+import { SemverVersioning } from './manual/SemverVersioning'
+import { IncrementalVersioning } from "./manual/IncrementalVersioning";
+import { BaseVersioning } from "./manual/BaseVersioning";
```

Comment:
I wonder why `manual` directory named as `manual` 🤔

Reply:
@floydkim coded without my brain 😅
I will refactor the code tomorrow